examples/archive.html

```diff
@@ -48,14 +50,13 @@ <h2 class="modal-title">Welcome to MapKnitter Lite</h2>
         </div>
       </div>
     </div>
-
-    <div id="map" style="width:100%; height:100%; position:absolute; top:0;"></div>
-    <i title="Open Sidebar" id="mapToggle" title="Open Sidebar" class="fa fa-bars fa-3x " style="position: absolute; right: 0; margin: 1rem; z-index: 900; color: white; cursor: pointer;" aria-hidden="true"></i>
-
+    
+    <i title="Open Sidebar" id="mapToggle" title="Open Sidebar" class="fa fa-bars fa-3x " style="position: absolute; right: 0; top:30px; margin: 1rem; z-index: 900; color: white; cursor: pointer;" aria-hidden="true"></i>
+    
     <div class="offcanvas offcanvas-end" data-bs-backdrop="false" data-bs-keyboard="false" tabindex="1" id="offcanvasRight" aria-labelledby="offcanvasRightLabel">
```

Comment:
Wow, it is very strange. I have to think maybe some of the click event handlers at the bottom of the file are interfering with this. 

But I also noticed that if I remove this whole 'offcanvas' div in the developer console, it then works. So... it's somehow involved?

Reply:
I can even set the offcanvas CSS `visibility: visible;` to `visibility: hidden;` and then the geocoding search box works. Mysterious!!

Reply:
That's very strange. @jywarren Since I am new to the community. Can you suggest me any other issue I can work with.

Reply:
Sure - @segun-codes and @7malikk are just starting their Outreachy fellowship and perhaps one of them may have an idea for a good issue? I'd also like to know what they think of this difficulty we're having. I would hope the fix is really simple, but we just need to identify the mechanism for the error. Thanks for your patience!

Reply:
Hey there @hs05june checkout this issue #1174, I believe it has not been claimed and is a good first issue for you. Thank you for your patience.
In the meantime, @jywarren, I will be looking into this bug and let you both know how it goes, my suspicions are the same as yours, the click events handlers could've posed some restrictions. Quite strange.

Reply:
@jywarren from the study I conducted, a `visibility: visible` style and a `show` class are appended on the offcanvas element when the offcanvas element is visible but said style is not removed when the `modalToggle` event listener is triggered. This is due to the setup of the said event listener which is an if statement that checks for the class `show` and removes it.

The solution to this problem would be:

- Take out the `modalToggle` event listener
```diff
-188  modalToggle.addEventListener('click', (event)=>{
-189            if(sidebar.classList.contains('show')){
-190             sidebar.classList.remove('show')
-191          }
-192   })
+188                                                                      
```

- Take out the close sidebar icon and remove the `d-sm-none` class from the `view map` button
```diff
-58    <i title="Close Sidebar" id="modalToggle" title="Close Sidebar" class=" fa fa-times fa-3x " style="cursor: pointer; position: absolute; right: 0; margin: 1rem; color: #00000070;" aria-hidden="true"></i>
+58                                                                               
-59   <button type="button" class="btn btn-primary d-sm-none" data-bs-dismiss="offcanvas" aria-label="Close">View Map</button>
+59   <button type="button" class="btn btn-primary " data-bs-dismiss="offcanvas" aria-label="Close">View Map</button>
```
- Take out the element declaration 
```diff
-88  let modalToggle = document.getElementById('modalToggle')
+88                                                             
```
- Finally, add some space to the open sidebar button to accommodate the geo search I used `top: 30px` and remove the duplicate `title` attribute
```diff
-53   <i title="Open Sidebar" id="mapToggle" title="Open Sidebar" class="fa fa-bars fa-3x " style="position: absolute; right: 0; margin: 1rem; z-index: 900; color: white; cursor: pointer;" aria-hidden="true"></i>
+53   <i title="Open Sidebar" id="mapToggle" class="fa fa-bars fa-3x " style="position: absolute; top: 30px; right: 0; margin: 1rem; z-index: 900; color: white; cursor: pointer;" aria-hidden="true"></i>
```

@jywarren I could convert this into an FTO after your review of course.

Reply:
> Hey there @hs05june checkout this issue #1174, I believe it has not been claimed and is a good first issue for you. Thank you for your patience. In the meantime, @jywarren, I will be looking into this bug and let you both know how it goes, my suspicions are the same as yours, the click events handlers could've posed some restrictions. It's quite strange.

@jywarren, I will take a look from here too

Reply:
> That's very strange. @jywarren Since I am new to the community. Can you suggest me any other issue I can work with.

Hi @hs05june, you may find this issue #855 interesting.

Reply:
Really great problem solving, and excellent suggested fix. Thanks all! @hs05june if you'd like to complete this we'd love that. But let us know and if we don't hear back in a day or two we can also add an extra commit to this PR to complete it. Thanks for your patience with a task that ended up a bit tougher than we'd expected!

Reply:
It works!!! Great work everyone!